tests/shared/src/test/scala/cats/tests/CatsEquality.scala

```diff
@@ -0,0 +1,29 @@
+package cats
```

Comment:
I think this whole file would be useful in tests that use the cats library, and not just cats itself. So, if that is the case, consider moving the file to `main`, so it could be exported - i.e. `tests/shared/src/main/scala/cats/tests/CatsEquality.scala`